pkg/networkservice/utils/inject/injectconncontext/client.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package injectconncontext changes connection context in client chain
+package injectconncontext
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"google.golang.org/grpc"
+
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+
+	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
+)
+
+type setConCtxClient struct {
+	context *networkservice.ConnectionContext
+}
+
+// NewSetConnectionContextClient - returns client that changes connection context to one, provided as parameter or default if nil provided
+func NewSetConnectionContextClient() networkservice.NetworkServiceClient {
```

Comment:
NewSetConnectionContextClient
```suggestion
func NewClient() networkservice.NetworkServiceClient {
```